### PR TITLE
Add explanation of micro resource

### PR DESCRIPTION
**Pull request by yoshwata**
Micro resource is available now, so I add the documentation.

reference:
https://github.com/screwdriver-cd/executor-k8s/pull/79
https://github.com/screwdriver-cd/queue-worker/pull/52
https://github.com/screwdriver-cd/executor-k8s-vm/pull/30
https://github.com/screwdriver-cd/screwdriver/pull/1019

### CODE REVIEW

**Review by yuichi10 — APPROVED** [review]
LGTM

File: docs/user-guide/configuration/annotations.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Some Annotations are used to modify the properties of the build environment. The
 | Annotation | Values | Description |
 |------------|--------|-------------|
 | beta.screwdriver.cd/executor | Ask your cluster admin | This will determine what compute system is used to run the build. For example, set the build to run in a VM, a kubernetes pod, a docker container, or a Jenkins agent. |
-| beta.screwdriver.cd/cpu | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 CPU resources (`LOW`) and 6 CPU resources (`HIGH`). Default is `LOW`. |
-| beta.screwdriver.cd/ram | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 GB RAM (`LOW`) and 12 GB RAM (`HIGH`). Default is `LOW`. |
+| beta.screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`. |
```
Comment (tk3fftk) [inline docs/user-guide/configuration/annotations.md]: I think "between" should be "among".

Comment (tk3fftk) [inline docs/user-guide/configuration/annotations.md]: >  When using a `k8s-vm` executor,

We can also use these annotations with `k8s` executor so we should also add the description.

**Review by minzcmu — APPROVED** [review]
👍 LGTM

File: docs/user-guide/configuration/annotations.md (hunk 1 above)
Comment (yoshwata) [reply docs/user-guide/configuration/annotations.md]: I think both is OK, but between is better in this scene.
https://eikaiwa.weblio.jp/column/study/eigo-bunpo/between-among
http://nagatuduki-eikaiwa.com/9592.html
https://www.englishcafe.jp/answer/anz-1c.html

Comment (yoshwata) [reply docs/user-guide/configuration/annotations.md]: I add `k8s` description. Thank you!

Comment (tk3fftk) [reply docs/user-guide/configuration/annotations.md]: it looks duplicate. I think we may not add lines, just add the sentence of `k8s` executor to existing lines.

Comment (tk3fftk) [reply docs/user-guide/configuration/annotations.md]: 🙆

Comment (yoshwata) [reply docs/user-guide/configuration/annotations.md]: I understood. I fixed.

**Review by s-yoshika — APPROVED** [review]
LGTM

**Review by tkyi — APPROVED** [review]

**Pull request merged**